corehq/apps/locations/tests/test_permissions.py

```diff
@@ -243,3 +241,57 @@ def test_user_can_acces_case(self):
                                            owner_id=self.locations['Cambridge'].location_id)
         self.assertTrue(user_can_access_case(self.domain, self.cambridge_worker, self.case))
         self.assertFalse(user_can_access_case(self.domain, self.suffolk_user, self.case))
+
+
+class TestLocationExport(LocationHierarchyTestCase):
+    domain = 'test-location-export'
+    location_type_names = ['state', 'county', 'city']
+    stock_tracking_types = ['state', 'county', 'city']
+    location_structure = [
+        ('England', [
+            ('Cambridgeshire', [
+                ('Cambridge', []),  # The other Cambridge ;)
+                ('Peterborough', []),
+            ]),
+            ('Lincolnshire', [
+                ('Boston', []),  # The other Boston ;)
```

Comment:
Apparently, the murder capital of England and Wales! 😨

Reply:
Apparently the numbers look bad because it's a small town. 

> [The data showed](https://www.gov.uk/government/statistics/crime-outcomes-in-england-and-wales-2014-to-2015) there were two homicides and eight attempted murders in the port town for the period up to September 2015 - meaning there is a rate of 15 murderous crimes for every 100,000 people.
> --- [The Independent, 2016-01-23](https://www.independent.co.uk/news/uk/crime/boston-in-lincolnshire-is-the-most-murderous-place-in-england-and-wales-home-office-figures-reveal-a6828571.html)

Two homicides? So this is where I come from:

> An annual ranking of the world’s 50 most dangerous cities drawn up by the Citizen Council for Public Safety and Criminal Justice, based in Mexico City, listed Cape Town as 11th last year, with 2,868 murders. The top ten were all in Mexico, Venezuela or Brazil.
> --- [The Times, 2019-07-01](https://www.thetimes.co.uk/article/soaring-murder-rate-drives-cape-town-up-list-of-deadliest-cities-xhqw3k0ld)